tests/test_instrument.py

```diff
@@ -51,7 +52,7 @@ def setUp(self):
         self.configFile = os.path.join(self.testDir, "config/basic/butler.yaml")
```

Comment:
Should we have a default file for `Butler` too perhaps?

Reply:
If only for users to copy. But it might be useful to do `butler = Butler()` and simply get something working with a local `SqliteRegistry` and `PosixDatastore` in a default local subdirectory.

Reply:
Perhaps even more useful if that then can write the default `butler.yaml` config in it (something like `git init .`) that you can subsequently modify.

Reply:
But maybe the default constructor is not the best for that. And instead you want an explicit class method e.g. `butler = Butler.makeDefault()` (with an optional path).